createHandler.js

```diff
@@ -200,7 +202,11 @@ export default function createHandler(
     _attachGestureHandler = newViewTag => {
       this._viewTag = newViewTag;
 
-      RNGestureHandlerModule.attachGestureHandler(this._handlerTag, newViewTag);
+      RNGestureHandlerModule.attachGestureHandler(
+        this._handlerTag,
+        newViewTag,
+        Platform.OS === 'web' ? this._propsRef : undefined
+      );
```

Comment:
the ios ci build failed, I wonder if you do need to separate the API calls by the platform like in your earlier commit...